tests/pdarray_creation_test.py

```diff
@@ -419,6 +419,12 @@ def test_full(self):
         string_len_full = ak.full("5", 5)
         self.assertEqual(5, len(string_len_full))
 
+        strings_full = ak.full(5, "test")
+        self.assertIsInstance(strings_full, ak.Strings)
+        self.assertEqual(5, len(strings_full))
+        for i in range(len(strings_full)):
+            self.assertEqual(strings_full[i], "test")
```

Comment:
Just noting that testing is one of the few cases where using `to_ndarray` and `to_list` is valid. This will test exactly what was request, so no changes are needed (purely for informational purposes), but we can also do something like below:

```python
self.assertListEqual(strings_full.to_list(), ["test"] * 5)
```